ui/pages/Address.tsx

```diff
@@ -295,8 +299,32 @@ const AddressPageContent = () => {
         undefined,
       ...formatUserTags(addressQuery.data),
       ...(addressMetadataQuery.data?.addresses?.[hash.toLowerCase()]?.tags.filter(tag => tag.tagType !== 'note') || []),
+      !addressQuery.data?.is_contract && xScoreFeature.isEnabled && xStarQuery.data?.data ?
```

Comment:
Shouldn't we check that it is not the placeholder data? Or the additional condition should be added to `<EntityTags/>` component usage.